### ISSUE TITLE
y-axis crosshair not working in some cases

### ISSUE THREAD

**Issue by pdanford** [highfestiva/finplot#145]
finplot v1.6
macOS Big Sur (11.2.3)
python verison 3.9.2

The following creates a small candlestick chart. The x-axis crosshair works correctly, but the y-axis value is stuck at 0:

```
import finplot as fplt
import pandas as pd

ax = fplt.create_plot("y-axis crosshair not working", rows=1, maximize=False)

data = [["2020-08-01T00:05:00", 10377.9, 11377.9, 10477.9, 11177.9, 28.6],
        ["2020-08-02T00:05:00", 10377.9, 11377.9, 10477.9, 11177.9, 28.6],
        ["2020-08-03T00:05:00", 10377.9, 11377.9, 10477.9, 11177.9, 28.6],
        ["2020-08-04T00:05:00", 10377.9, 11377.9, 10477.9, 11177.9, 28.6],
        ["2020-08-05T00:05:00", 10377.9, 11377.9, 10477.9, 11177.9, 28.6],
        ["2020-08-06T00:05:00", 10377.9, 11377.9, 10477.9, 11177.9, 28.6],
        ["2020-08-07T00:05:00", 10377.9, 11377.9, 10477.9, 11177.9, 28.6],
        ["2020-08-08T00:05:00", 10377.9, 11377.9, 10477.9, 11177.9, 28.6],
        ["2020-08-09T00:05:00", 10377.9, 11377.9, 10477.9, 11177.9, 28.6],
        ["2020-08-10T00:05:00", 10377.9, 11377.9, 10477.9, 11177.9, 28.6],
        ["2020-08-11T00:05:00", 10377.9, 11377.9, 10477.9, 11177.9, 28.6],
        ["2020-08-12T00:05:00", 10377.9, 11377.9, 10477.9, 11177.9, 28.6],
        ["2020-08-13T00:05:00", 10377.9, 11377.9, 10477.9, 11177.9, 28.6],
        ["2020-08-14T00:05:00", 10377.9, 11377.9, 10477.9, 11177.9, 28.6],
        ["2020-08-15T00:05:00", 10377.9, 11377.9, 10477.9, 11177.9, 28.6],
        ["2020-08-16T00:05:00", 10377.9, 11377.9, 10477.9, 11177.9, 28.6],
        ["2020-08-17T00:05:00", 10377.9, 11377.9, 10477.9, 11177.9, 28.6],
        ["2020-08-18T00:05:00", 10377.9, 11377.9, 10477.9, 11177.9, 28.6],
        ["2020-08-19T00:05:00", 10377.9, 11377.9, 10477.9, 11177.9, 28.6],
        ["2020-08-20T00:05:00", 10377.9, 11377.9, 10477.9, 11177.9, 28.6],
        ["2020-08-21T00:05:00", 10377.9, 11377.9, 10477.9, 11177.9, 28.6]]

candles_df = pd.DataFrame(data, columns = "time low high open close vol".split())
candles_df = candles_df.astype({"time":"datetime64[ns]"})

candles = candles_df["time open close high low".split()]
fplt.candlestick_ochl(candles, ax=ax)

fplt.show()
```

**Comment by highfestiva** [comment]
Caused by a bug in the calculation of significant digits, fixed in 4e6f69412a4f75a8c980d3d23e17096c4b9138b6.

**Comment by pdanford** [comment]
Thanks! And thank you for finplot and its documentation. Really nice work!
